doc/tutorials/enable_s5.rst

```diff
@@ -31,131 +31,197 @@ The diagram below shows the overall architecture:
+
+.. note:: Service VM is always allowed to trigger system S5 by default.
 
 .. _enable_s5:
 
 Enable S5
 *********
 
-The procedure for enabling S5 is specific to the particular OS:
-
-* For Linux (LaaG) or Windows (WaaG), include these lines in the launch script:
-
-  .. code-block:: bash
+1. Configure communication vUART for Service VM and User VMs:
+
+   Add these lines in the hypervisor scenario XML file manually:
+
+   Example::
+
+      /* VM0 */
+      <vm_type>SERVICE_VM</vm_type>
+      ...
+      <legacy_vuart id="1">
+         <type>VUART_LEGACY_PIO</type>
+         <base>CONFIG_COM_BASE</base>
+         <irq>0</irq>
+         <target_vm_id>1</target_vm_id>
+         <target_uart_id>1</target_uart_id>
+      </legacy_vuart>
+      <legacy_vuart id="2">
+        <type>VUART_LEGACY_PIO</type>
+        <base>CONFIG_COM_BASE</base>
+        <irq>0</irq>
+        <target_vm_id>2</target_vm_id>
+        <target_uart_id>2</target_uart_id>
+      </legacy_vuart>
+      ...
+      /* VM1 */
+      <vm_type>POST_STD_VM</vm_type>
+      ...
+      <legacy_vuart id="1">
+         <type>VUART_LEGACY_PIO</type>
+         <base>COM2_BASE</base>
+         <irq>COM2_IRQ</irq>
+         <target_vm_id>0</target_vm_id>
+         <target_uart_id>1</target_uart_id>
+      </legacy_vuart>
+      ...
+      /* VM2 */
+      <vm_type>POST_STD_VM</vm_type>
+      ...
+      <legacy_vuart id="1">
+         <type>VUART_LEGACY_PIO</type>
+         <base>INVALID_COM_BASE</base>
+         <irq>COM2_IRQ</irq>
+         <target_vm_id>0</target_vm_id>
+         <target_uart_id>2</target_uart_id>
+      </legacy_vuart>
+      <legacy_vuart id="2">
+         <type>VUART_LEGACY_PIO</type>
+         <base>COM2_BASE</base>
+         <irq>COM2_IRQ</irq>
+         <target_vm_id>0</target_vm_id>
+         <target_uart_id>2</target_uart_id>
+      </legacy_vuart>
+      ...
+      /* VM3 */
+      ...
+
+  .. note:: These vUART is emulated in the hypervisor; expose the node as ``/dev/ttySn``.
+     For the first User VM, the communication vUART index should be 1. For other User VMs,
+     the vUART (index is 1) shoulbe be configured as invalid, the communication vUART index
+     should be 2 or others.
+  .. note:: serial.conf file will be generated for Service VM if above communication vUARTs
+     are configured, need to copy ``build/hypervisor/serial.conf`` into ``/etc`` folder of Service VM.
+  .. note:: Need to install ``setserial`` tool in Service VM.
+
+2. Build the Lifecycle Manager daemon, ``life_mngr``:
 
-     # Power Management (PM) configuration using vUART channel
-     pm_channel="--pm_notify_channel uart"
-     pm_by_vuart="--pm_by_vuart pty,/run/acrn/life_mngr_"$vm_name
-     pm_vuart_node="-s 1:0,lpc -l com2,/run/acrn/life_mngr_"$vm_name
+   .. code-block:: none
 
-     acrn-dm -A -m $mem_size -s 0:0,hostbridge \
-       ...
-       $pm_channel \
-       $pm_by_vuart \
-       $pm_vuart_node \
-       ...
+      $ cd acrn-hypervisor
```

Comment:
```suggestion
      cd acrn-hypervisor
```

Reply:
It's best to not include the prompt (e.g. `$` or `#`) so people can more easily copy-paste the instructions.

Can you remove them in all other instances in this PR?

Reply:
Thanks for your comments, will update this in the whole document.